snuba_sdk/query.py

```diff
@@ -47,8 +47,8 @@ class Query:
     match: Entity
     select: Optional[List[Union[Column, CurriedFunction, Function]]] = None
     groupby: Optional[List[Union[Column, CurriedFunction, Function]]] = None
-    where: Optional[List[Condition]] = None
-    having: Optional[List[Condition]] = None
+    where: Optional[List[Union[BooleanCondition, Condition]]] = None
+    having: Optional[List[Union[BooleanCondition, Condition]]] = None
```

Comment:
I think you may want to make this a `Condition`. If there is an `And` condition I don't see a strong need to also allow a List of `Condition` which mean the same thing by introducing some minimal overhead.
IT would make processing and readability easier as people would not have to remember the meaning of a sequence of conditions.

Reply:
You mean to make this the same as the Snuba AST? Where there is one base condition that holds all the others?

Reply:
yes

Reply:
That will unnecessarily clutter up the interface I think, since you'll have to use `And`/`Or` even if you only have one condition. I'm not sure why it would be more readable.